deb-get

```diff
@@ -113,6 +113,9 @@ function install_apt() {
     if [ -n "${APT_KEY_URL}" ]; then
         wget -q "${APT_KEY_URL}" -O "/etc/apt/trusted.gpg.d/${APT_LIST_NAME}.asc"
     fi
+    if [ -n "${APT_KEY_ID}" ]; then
+        apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys "${APT_KEY_ID}"
```

Comment:
apt-key is deprecated. So an alternative method of acquiring the key is required.

Reply:
okie, didn't realise it was deprecated. Updated PR, with updated URL. tested this in a LXD container, and seems to work for me